controllers/suite_test.go

```diff
@@ -86,18 +88,18 @@ func TestMain(m *testing.M) {
 		}).SetupWithManager(ctx, mgr, controller.Options{MaxConcurrentReconciles: 1}); err != nil {
 			panic(fmt.Sprintf("Failed to start ClusterCacheReconciler: %v", err))
 		}
-		if err := (&ClusterReconciler{
+		if err := (&cluster.Reconciler{
 			Client:    mgr.GetClient(),
 			APIReader: mgr.GetClient(),
-			recorder:  mgr.GetEventRecorderFor("cluster-controller"),
+			Recorder:  mgr.GetEventRecorderFor("cluster-controller"),
```

Comment:
Looks like the recorder is later overwritten in `SetupWithManager` so I think we don't have to set & export the field here (given that there are no other usages outside of the package of the reconciler)

(same for machine.Reconciler)

Reply:
fixed